CHANGES.md

```diff
@@ -3,4 +3,8 @@ Change Log
 
 ### 0.0.0 - 2020-11-12
 
-* Initial release
\ No newline at end of file
+* Initial release
+
+### 0.1.0 - 2020-11-14
```

Comment:
```suggestion
### 0.0.1 - 2020-??-??
```

Since this is a bug fix increment the patch version instead of the minor version. Also since we don't know the release date yet we usually just add ?? until someone does the release.